source/javascripts/_apihandler.js.erb

```diff
@@ -1,11 +1,13 @@
+var isPluginMode = "<%= mode %>" == "cli" && "<%= build? %>" == "true";
```

Comment:
how about `var isPluginMode = <%= mode == "cli" && build %>`?

Reply:
Good point, my solution still results in ugly looking static code. With this one it looks ok.